cruise-control/src/test/java/com/linkedin/kafka/cruisecontrol/analyzer/AnalyzerUnitTestUtils.java

```diff
@@ -54,4 +54,15 @@ public static Goal goal(Class<? extends Goal> goalClass, Properties configOverri
   public static Goal goal(Class<? extends Goal> goalClass) throws Exception {
     return goal(goalClass, new Properties());
   }
+
+  /**
+   * Reflectively create goal object from specified goal class and a set of config overrides.
+   *
+   * @param goalClass The goal class to create object.
+   * @param configOverrides A set of config properties to override.
+   * @return New object of specified class.
+   */
+  public static Goal goalWithConfigOverrides(Class<? extends Goal> goalClass, Properties configOverrides) throws Exception {
+    return goal(goalClass, configOverrides);
+  }
```

Comment:
What is the purpose of this new function?
Why don't the caller of this function just call `goal(goalClass, configOverrides);`?

Can we drop this function?

Reply:
Yes, we can drop it